### PR TITLE
Fixed code indentation in ParquetCachedBatchSerializer

### PR DESCRIPTION
**Pull request by razajafri**
Signed-off-by: Raza Jafri <rjafri@nvidia.com>

### CODE REVIEW

File: shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala [hunk 1]
```diff
@@ -884,15 +885,16 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
...
           for (i <- 0 until columns.size) {
             if (!missingColumns(i)) {
               columnReaders(i) =
-                new VectorizedColumnReader(columns.get(i), types.get(i)
-                  .getOriginalType, pages.getPageReader(columns.get(i)), null /*convertTz*/,
-                  LegacyBehaviorPolicy.CORRECTED.toString, LegacyBehaviorPolicy.EXCEPTION.toString)
+                  new VectorizedColumnReader(columns.get(i), types.get(i)
+                      .getOriginalType, pages.getPageReader(columns.get(i)), null /*convertTz*/ ,
+                    LegacyBehaviorPolicy.CORRECTED.toString,
+                    LegacyBehaviorPolicy.EXCEPTION.toString)
```
Comment (jlowe) [inline shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala:897]: If we're already wrapping on multiple lines for the number of arguments, I don't think it's worth splitting an argument across lines that can fit on a line.  Personally I'd put one arg per line to make it easier to line up the arguments (both in terms of indentation and in terms of parameter ordering), e.g.:
```suggestion
                  new VectorizedColumnReader(
                    columns.get(i),
                    types.get(i).getOriginalType,
                    pages.getPageReader(columns.get(i)),
                    null /*convertTz*/ ,
                    LegacyBehaviorPolicy.CORRECTED.toString,
                    LegacyBehaviorPolicy.EXCEPTION.toString)
```

**Review by jlowe — APPROVED** [review]
Noticed this is a draft PR, but there's no comment explaining why?

**Comment by razajafri** [comment]
I had made it a draft because of https://github.com/NVIDIA/spark-rapids/pull/2459 PR. But now that I have to rework https://github.com/NVIDIA/spark-rapids/pull/2459. Its fine for this to go in first, I will change it

**Comment by razajafri** [comment]
build

**Pull request merged**